generators/spa/templates/Vue3JS/common/obj/Release/UpendoVentures.UpendoSendUI.csproj.CoreCompileInputs.cache

```diff
@@ -0,0 +1 @@
+0bc7793b7e22ae5cc80ca1f786f579c15002d7a3
```

Comment:
@alejoroman0605 This file should not be committed.

Reply:
@alejoroman0605 You ignored this comment.  CC: @ismaelTartola

Reply:
@WillStrohl I just fixed it https://github.com/UpendoVentures/generator-upendodnn/pull/280/commits/b667ca74420311f604a3735b866d4c842e9b08e0